docs/windows.md

```diff
@@ -149,14 +149,16 @@ kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/
 
 #### Join Windows worker Nodes
 
-#### 1. (Optional, Test-Only) Install OVS provided by Antrea
+#### 1. (Optional) Install OVS provided by Antrea
 
 Antrea provides a pre-built OVS package which contains test-signed OVS kernel
 driver. If you don't have a self-signed OVS package and just want to try the
 Antrea on Windows, this package can be used for testing. We also provide a help
-script to install the OVS driver and register userspace binaries as services.
+script `Install-OVS.ps1` to install the OVS driver and register userspace binaries
+as services. If you want to use your own signed OVS package for production, you can
+run `Install-OVS.ps1` like this: `Install-OVS.ps1 -ImportCertificate $false -Local -LocalFile <PathToOVSPackage>`
```

Comment:
nit: for the full command use a code block with a `powershell` code specifier

Reply:
Updated.